src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -95,6 +102,20 @@ class IOUAmountPage extends React.Component {
         this.focusTextInput();
     }
 
+    componentWillUnmount() {
+        if (!this.unsubscribeAppStateSubscription) {
+            return;
+        }
+        this.unsubscribeAppStateSubscription();
+    }
+
+    handleAppStateChange(nextAppState) {
```

Comment:
Methods need to be named for what they do rather than for what they handle. I suggest renaming this to `dismissKeyboardWhenBackgrounded`